@commitlint/lint/src/lint.test.ts

```diff
@@ -283,3 +283,23 @@ test('returns original message with commit header, body and footer, parsing comm
 
 	expect(report.input).toBe(expected);
 });
+
+test('passes for async rule', async () => {
+	const report = await lint(
+		'somehting #1',
+		{
+			'async-rule': [2, 'never']
+		},
+		{
+			plugins: {
+				'example-plugin': {
+					rules: {
+						'async-rule': async () => [true, 'all good'] as const
```

Comment:
can you change this test to have any timeout

```suggestion
						'async-rule': () =>
							new Promise(resolve => {
								setTimeout(() => resolve([true, 'all good'] as const), 5);
							})
```

as `config-lerna-scopes` already has async and it seems to be working.